src/links/links.controller.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpException,
+  HttpStatus,
+  NotFoundException,
+  Param,
+  Patch,
+  Post,
+} from '@nestjs/common';
+import { LinksService } from './links.service';
+import { Link, Prisma } from '@prisma/client';
+
+@Controller('links')
+export class LinksController {
+  constructor(private readonly linksService: LinksService) {}
+
+  @Get() // GET   /links
+  async getAllLinks() {
```

Comment:
**praise**: Good use of try-catch blocks to handle potential errors in controller methods.